CONTRIBUTING.md

```diff
@@ -76,15 +76,50 @@ To address feedback from code review, implement changes and push to your feature
 ### Code style and formatting
 The Lingua Franca compiler is implemented in Java and Kotlin. The overarching advice is to use each language's most widely used idioms and conventions, which are fortunately very similar. The code base is shipped with a [Spotless](https://github.com/diffplug/spotless) configuration to check and enforce style compliance. Lingua Franca code (e.g., tests) in this repository is also automatically formatted via Spotless.
 
+
+#### Formatting
+
 - To check that modified files are formatted correctly, run:
+
 ```
 ./gradlew spotlessCheck
 ```
+
 - To apply the changes recommended by the formatter, run:
+
 ```
 ./gradlew spotlessApply
```

Comment:
I've always wondered why `./gradlew spotlessCheck` suggests to run `./gradlew :spotlessApply`. What is the `:` for?

Reply:
I do not know the details, but I believe the colon `:` is used in Gradle to delimit segments of hierarchical namespaces. So, maybe starting with a colon means that something is in the root namespace.

I think you get the same results either way.

Reply:
Yes, you can also use `./gradlw cli:lfc:spotlessApply` to only format the lfc subproject. This works for all gradle tasks including `build`, `test`, `clean` etc.. If you run a task without a qualifier, then gradle will search for tasks with this name in all subprojects and execute them all.